package.json

```diff
@@ -28,19 +28,19 @@
   },
   "dependencies": {
     "lodash.get": "^4.4.2",
-    "lodash.set": "^4.3.2",
-    "redux-persist": "^3.4.0"
+    "lodash.set": "^4.3.2"
   },
   "devDependencies": {
     "babel-cli": "^6.11.4",
     "babel-preset-es2015": "^6.13.2",
     "babel-register": "^6.11.6",
     "chai": "^3.5.0",
     "mocha": "^3.0.2",
-    "release-it": "^2.4.3"
+    "release-it": "^2.4.3",
+    "redux-persist": "^4.0.1"
```

Comment:
I moved redux-persist to dev deps to allow the consumer app to use v3 or v4 without pulling in both versions.